### ISSUE TITLE
ssh-agent not working as expected.

### ISSUE THREAD

**Issue by Offerel** [keepassxreboot/keepassxc#8078]
I didn’t get KeepassXC integration to work as expected. I have created a systemd unit as described in the documentation at https://ferrario.me/using-keepassxc-to-manage-ssh-keys/. It seems that this systemd service is starting. At least i see a ssh-agent process running. But after autostarting KeepassXC, there is a red exclamation, that KeepassXC cant find the running process. If i restart the systemd service when KeepassXC is running (`systemctl restart --user ssh-agent`) the ssh-agent is also found by KeepassXC. Is there a better way to archive ssh-agent integration?

I'm on Gnome in Manjaro.

**Comment by hifi** [comment]
It's highly likely you already have an ssh agent running if you're on GNOME or you can enable it from GNOME Keyring.

Have you checked these?

Thanks.

**Comment by Offerel** [comment]
Yes, I have checked that. There is no ssh-agent from Manjaro itself. I have disabled all other ssh-agent starts as described in the tutorial. The command I entered later do only restart the already running instance.

I have never used a systemd service under --user option, and as such don't know how that works in detail. But I assume that the service starts before KeepassXC and with that KeepassXC didn't find the process. But maybe I'm wrong with that.

Anyone with more experience as me, with systemd knowledge should investigate this. Maybe (s)he has a idea what I do wrong. Im sure it's my own fault or misknowledge.

Btw: speaking of Gnome-keyring. This is disabled also, because I want to have the secret service integration running. Maybe this is the main problem and I have to use some other way in that case.
